undead-discord.css

```diff
@@ -295,7 +295,23 @@
     mask: none !important;
 }
 
+/* Hides mask for small pictures in LFG pane */
+.avatarMasked-1DKyfL, .emptyUser-34aCrC {
+  -webkit-mask-size: initial;
+  mask-size: initial;
+}
+
 /* This is for the new title bar on Windows, adding a border under it */
 .titleBar-3_fDwJ {
   border-bottom: 1px solid var(--darkest-bg-color);
 }
+
+/* Handles server cards so hovering doesn't cause background to not match theme */
+.theme-dark .card-11ynQk:before {
+  opacity: 0;
```

Comment:
Does a `display: none` work for this too? Or does this fuck with other things? Just making it invisible seems like an odd choice. There's also `visibility: hidden` if you really just want to make it invisible. Don't think opacity is the best choice for this.

Reply:
Hm, works with `display: none` so i just moved it to that block. I used `opacity: 0` because that's what was shown in the css file. I moved the classes do the display none block and removed the opacity block

Reply:
Yeah that sounds great, thanks!